source/clear-linux/tutorials/kata_migration.rst

```diff
@@ -14,26 +14,28 @@ To switch to Kata Containers in |CL|, complete the following:
 Stop Clear Containers instances
 *******************************
 
-Before doing any change, all running Clear Containers must be stopped:
+Before making any changes, all running Clear Containers must be stopped:
 
 .. code-block:: bash
 
-    $ for container in $(sudo docker ps -q); do sudo docker stop $container; done
+    for container in $(sudo docker ps -q); do sudo docker stop $container; done
```

Comment:
Hey folks, isn't our agreement that if we remove the '$', we should then add "As an unprivileged user..." ?